ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -31,6 +31,7 @@ export default class TestDataModal extends React.Component {
         'mozilla-central',
       ),
       platform: 'linux64',
+      pinnedProjects: ['autoland', 'mozilla-central', 'mozilla-beta', 'try'],
```

Comment:
I think that `pinnedItems` would make a better, more generic name, since pinned items can be either projects, platforms, frameworks or statuses.

Reply:
Previously I had a dictionary called `pinned` with the keys `projects`, `platforms`, and `frameworks` and was told it was unnecessary

Reply:
it can't simply be `pinnedItems` as this state is shared and we need to differentiate between what type of pinned item it is